README.md

```diff
@@ -55,41 +55,78 @@ export default config;
 The `storybook-builder-wds` aims at compatibility with standard storybook features, e.g. `.storybook/preview.js`.
 Follow the [official storybook docs](https://storybook.js.org/) for the configuration options.
 
-### Configuring @web/dev-server
+### Configuring dev server
 
-The builder comes with a default `@web/dev-server` configuration which adheres to browser standards and supports essential plugins of the storybook ecosystem.
+The builder implements `storybook dev` command and comes with a default `@web/dev-server` configuration which adheres to browser standards and supports essential plugins of the storybook ecosystem.
 
 When storybook is loaded, the default config gets automatically merged with the project configuration file (`web-dev-server.config.{mjs,cjs,js}`) if present.
 This is needed to ensure that the same configuration is used for application, feature or design system you are building.
 
 #### wdsFinal hook
 
-Sometimes you might need to add a storybook specific configuration.
+Sometimes you might need to add a storybook specific configuration for dev server.
 There is `wdsFinal` hook for that.
 
 ```js
 // .storybook/main.js
-import { mergeConfigs } from '@web/dev-server';
-
 /** @type { import('storybook-builder-wds').StorybookConfigWds } */
 const config = {
   ...
   core: {
     builder: 'storybook-builder-wds',
   },
   async wdsFinal(config) {
-    return mergeConfigs(config, {
-      // add storybook specific configuration for @web/dev-server
-      // e.g. "open" to go to a custom URL in the browser when server has started
-      open: '/custom-path',
-    });
+    // add storybook specific configuration for @web/dev-server
+    // e.g. "open" to go to a custom URL in the browser when server has started
+    config.open = '/custom-path';
+    return config;
+  },
+  ...
+};
+
+export default config;
+```
+
+The `config` parameter is a `@web/dev-server` config, make sure to use appropriate options and @web/dev-server plugins.
+When using rollup plugins make sure to [convert them to @web/dev-server ones](https://modern-web.dev/docs/dev-server/plugins/rollup/).
+
+### Configuring static build
+
+The builder implements `storybook build` command and comes with a default `rollup` configuration which adheres to browser standards and supports essential plugins of the storybook ecosystem.
+
+It's using [@web/rollup-plugin-html](https://modern-web.dev/docs/building/rollup-plugin-html/) under the hood.
```

Comment:
Im not sure this is relevant documentation for a consumer

Reply:
@thepassle I saw it had some specific integration between html and other plugins, at least this one https://modern-web.dev/docs/building/rollup-plugin-polyfills-loader/#multiple-rollup-build-outputs, so I thought it would be nice to link to that page, but indeed it's not super relevant, I'll remove it